packages/lumx-react/src/components/button/IconButton.tsx

```diff
@@ -61,7 +61,13 @@ export const IconButton: Comp<IconButtonProps, HTMLButtonElement> = forwardRef((
 
     return (
         <Tooltip label={hideTooltip ? '' : label} {...tooltipProps}>
-            <ButtonRoot ref={ref} {...{ emphasis, size, theme, ...forwardedProps }} aria-label={label} variant="icon">
+            <ButtonRoot
+                ref={ref}
+                {...{ emphasis, size, theme, ...forwardedProps }}
+                aria-label={label}
+                variant="icon"
+                aria-describedby={tooltipProps?.label && tooltipProps?.label !== label ? null : undefined}
```

Comment:
I'm not sure why we need to do this? 
Why the distinction between null and undefined? Is it because it will be overridden afterwards and "null" completely removes it while undefined doesn't?
Could you add a comment explaining what's happening?

Reply:
`null` was not useful here, I removed it and added a comment